packages/devtools_app/lib/src/shared/banner_messages.dart

```diff
@@ -25,13 +26,13 @@ const preCompileShadersDocsUrl =
 
 class BannerMessagesController {
   final _messages = <String, ValueNotifier<List<BannerMessage>>>{};
-  final _dismissedMessageKeys = <Key>{};
+  final Set<Key?> _dismissedMessageKeys = <Key?>{};
```

Comment:
why do we need to add the type on the LHS? `<Key?>` should be enough

Reply:
done